pkgs/development/python-modules/glad2/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, python3
```

Comment:
Not okay in `pythonPackages`. This would use Python 3.10 for `python311Packages.glad2`. Fixed in https://github.com/NixOS/nixpkgs/pull/244135